src/index.js

```diff
@@ -1,64 +1,67 @@
 import './style.css';
+import NewTodo from '../modules/crud.js';
+import createTodo from '../modules/create.js';
 
-const todoUl = document.getElementById('todoList');
+const form = document.getElementById('form');
+export const todo = document.querySelector('.add_todo');
+export const todoUl = document.getElementById('todoList');
+export const todoListss = JSON.parse(localStorage.getItem('todos')) || [];
+// const deleteButton = document.getElementById('clear_button');
 
-const todoLists = [
-  {
-    id: 1,
-    description: 'Connect to microverse morning call',
-    completed: true,
-  },
-  {
-    id: 2,
-    description: 'connect to microverse stand up call',
-    completed: false,
-  },
-  {
-    id: 3,
-    description: 'Prepare breakfast',
-    completed: false,
-  },
-  {
-    id: 4,
-    description: 'watch a youtube video on webpack',
-    completed: true,
-  },
-  {
-    id: 5,
-    description: 'have my bath',
-    completed: true,
-  },
-];
+// const todoLists = [
+//   {
+//     id: 1,
+//     description: 'Connect to microverse morning call',
+//     completed: true,
+//   },
+//   {
+//     id: 2,
+//     description: 'connect to microverse stand up call',
+//     completed: false,
+//   },
+//   {
+//     id: 3,
+//     description: 'Prepare breakfast',
+//     completed: false,
+//   },
+//   {
+//     id: 4,
+//     description: 'watch a youtube video on webpack',
+//     completed: true,
+//   },
+//   {
+//     id: 5,
+//     description: 'have my bath',
+//     completed: true,
+//   },
+// ];
 
-const displayTodos = () => {
-  for (let i = 0; i < todoLists.length; i += 1) {
-    const item = todoLists[i];
-    const itemContainer = document.createElement('li');
-    itemContainer.className = 'todo';
-    const todoTitle = document.createElement('h3');
-    todoTitle.className = 'todo_title';
-    todoTitle.innerText = item.description;
-    const checkBox = document.createElement('input');
-    checkBox.type = 'checkbox';
-    checkBox.checked = item.completed;
-    checkBox.className = 'checkbox';
-    const option = document.createElement('span');
-    option.className = 'material-symbols-outlined';
-    option.innerText = 'more_vert';
-
-    const div = document.createElement('div');
-    div.appendChild(checkBox);
-    div.appendChild(todoTitle);
-    div.className = 'todo_right';
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const newTodo = new NewTodo(todo.value);
+  newTodo.add();
+  localStorage.setItem('todos', JSON.stringify(todoListss));
+  form.reset();
+});
 
-    itemContainer.appendChild(div);
-    itemContainer.appendChild(option);
+// functionality to clear all the todos
+// deleteButton.addEventListener('click', () => {
+//   deleteCompletedTodos();
+// });
 
-    todoUl.appendChild(itemContainer);
-  }
+export const displayTodos = () => {
+  todoListss.forEach((todo, ind) => {
+    todo.id = ind + 1;
+    createTodo(todo);
+  });
 };
 
+// localStorage.removeItem('todos');
```

Comment:
- [ ] Kindly remove commented-out code. They can make your code messy and more confusing for other developers and don't add functionality.